assets/device/classes/vacuumcleaner.json

```
@@ -18,7 +20,9 @@
     "sv": "Använd den här enhetsklassen för dammsugare, oftast tillsammans med funktionen `dammsugar_status`.",
     "no": "Bruk denne enhetsklassen for støvsugere, ofte sammen med funksjonen `støvsuger_status`.",
     "es": "Utiliza esta clase de dispositivo para las aspiradoras, generalmente junto con la capacidad 'estado_aspiradora'.",
-    "da": "Brug denne enhedsklasse til støvsugere, normalt sammen med `støvsugertilstand` funktionen."
+    "da": "Brug denne enhedsklasse til støvsugere, normalt sammen med `støvsugertilstand` funktionen.",
```

```suggestion
    "da": "Brug denne enhedsklasse til støvsugere, normalt sammen med `vacuumcleaner_state` funktionen.",
```
This should also be fixed for other translations in this description